TEMPLATE.md

```diff
@@ -1,4 +1,7 @@
-# <Name of Feature>
+# {Name of Feature}
+
+- Steward: {Your Name} {you@example.com}, {@you} on the Elm Slack
+- Questions + Collaboration: {room} on the Elm Slack
```

Comment:
You can create channels by request, but you have to ask in #admin-help. But since there are so many, there's likely already a relevant channel to discuss in. Or take it to #general.